fhir-resource-tooling/src/test/resources/example-cos-credentials.json

```diff
@@ -0,0 +1,6 @@
+{
+  "access_key_id": "<HMAC access_key_id from COS service credentials>",
+  "secret_access_key": "<HMAC secret_access_key from COS service credentials>",
```

Comment:
Did you think about whether or not this variable should be encrypted within the config file? It used to be that passwords were required to be encrypted in config files. With K8S and secrets, I am much less clear on the security requirements.